### ISSUE TITLE
Can't restore saved network

### ISSUE THREAD

**Issue by techtonik** [nodebox/nodebox#423]
I see the output, but graph pane is empty. =/ No output in terminal or console window.

[barbox.ndbx.zip](https://github.com/nodebox/nodebox/files/306871/barbox.ndbx.zip)

**Comment by fdb** [comment]
Try zooming out 😄 

![screen shot 2016-06-20 at 10 45 55](https://cloud.githubusercontent.com/assets/8477/16188109/30232cd8-36d4-11e6-93b1-7e05d7ac63bc.png)

**Comment by techtonik** [comment]
How to do this?

**Comment by techtonik** [comment]
Mouse-wheel helped. =) But it looks like it is a bug with Nodebox save ability.

**Comment by techtonik** [comment]
If you save the file and then reload it, zoom level will be reset and box will be lost again.

**Comment by fdb** [comment]
Yes — NodeBox doesn't keep the location of where you zoomed. If you have nodes that you move outside of the standard box they appear to be hidden. In your case the node is at location `1,-1`, so just off-screen.

A solution we do is to move the box back to the "reset" position. In your case it means panning the view a little bit "up" (so holding spacebar and dragging down), then moving the node down one or two squares.

**Comment by techtonik** [comment]
Move manually? Why don't adjust all coordinates to 0,0 coordinate on save?

(another bug - holding spacebar with down works only for half of the second, then everything freezes)

**Comment by techtonik** [comment]
Saved nodes can be panned on save, but it looks like writer is not the best place to do this.

https://github.com/nodebox/nodebox/blob/c15874a11f7b90c262afcd2db5f20ea69dc011b2/src/main/java/nodebox/node/NDBXWriter.java#L43

**Comment by fdb** [comment]
"Panning" on save sounds interesting. However, what happens if the bounding box of available nodes is bigger than that of the view? Do we just make sure that the top-left node is on 0,0?

**Comment by techtonik** [comment]
If bounding box is bigger, Nodebox still resets view to 0,0 on load, so placing top-left node there helps to unwind the graph after this jump.
